lib/pf/Switch/Meru/MC_v2.def

```diff
@@ -0,0 +1,14 @@
+description=Meru Controller v2
+radiusDisconnect=disconnect
+acceptVlan=<<EOT
+Tunnel-Medium-Type = 6
+Tunnel-Type = 13
+Tunnel-Private-Group-ID = $vlan
+EOT
+acceptRole=<<EOT
+Filter-ID = $role
+EOT
+disconnect=<<EOT
+Calling-Station-Id = $mac
+EOT
+
```

Comment:
Add the following comment, please.
Then I will commit.
```
# Do not edit.
# Any changes will be lost on upgrade.	
```

Reply:
Done